ui/components/app/wallet-overview/coin-buttons.tsx

```diff
@@ -389,11 +416,13 @@ const CoinButtons = ({
           )}
           <IconButton
             className={`${classPrefix}-overview__button`}
+            iconButtonClassName={iconButtonClassName}
             data-testid={`${classPrefix}-overview-receive`}
             Icon={
               <Icon
                 name={IconName.ScanBarcode}
                 color={IconColor.primaryInverse}
+                size={IconSize.Sm}
               />
             }
             label={t('receive')}
```

Comment:
non-blocking: Would be great to get a storybook file for this component so we can view and work on it in isolation

Reply:
Makes sense, added here [fa835b1](https://github.com/MetaMask/metamask-extension/pull/27161/commits/fa835b12d05ca8fe3994d5316913d9ed4e7b2bdc)